submissions/meduna oluwafemi/HTML/landingpage.html

```diff
@@ -0,0 +1,209 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.min.css" integrity="sha256-+N4/V/SbAFiW1MPBCXnfnP9QSN3+Keu+NlB+0ev/YKQ=" crossorigin="anonymous" />
+    <link rel="stylesheet" href="css/landingstyle.css">
+    <title>Document</title>
+</head>
+<body>
+
+    <div class="header">
+            <header>
+                
+        <nav>
+            <div class="logo">
+             <h4>sneaker<span>.hub</span></h4>
+            </div>
+                
+                <ul class="nav-middle">
+                    <li>
+                    <a href="#">recent arrivals</a>
+                    </li>
+                
+                    <li>
+                    <a href="#">Men</a>
+                    </li>
+                
+                    <li>
+                    <a href="#">Women</a>
+                    </li>
+                
+                    <li>
+                    <a href="#">Kids</a>
+                    </li>
+                
+                    <li>
+                    <a href="#">Accessories</a>
+                    </li>
+                        
+                </ul>
+                
+                <div class="search">
+                <i class="fas fa-search"></i>
+            </div>
+            
+                <div class="login">
+                       <a href="index.html" target="blank"><button id="login-button">LOGIN</button></a> 
+                           <i class="fas fa-user"><a href="#"></a></i>
+                            <i class="fas fa-shopping-cart"><a href="#"></a></i>
+                 </div>          
+            </nav>
+                
+            </header>  
+
+    </div>
+
+   
+<div class="banner">
+        <h2>"90's collection" </h2>
+        <p>authentic apparrel for Lorem ipsum dolor sit amet.</p>
+        <button class="explore">explore</button>
+</div>
+
+
+<div >
+    <p id="product-info">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Autem, culpa. 
+         Ipsam repellendus corrupti omnis, repudiandae autem commodi odio harum 
+         voluptatem maiores quod deleniti!  A quo esse quidem non eaque omnis iusto, 
+          officiis illum cum alias sunt quod obcaecati, ut sequi.</p>
+   
+          <div>
+                   <a href="productpage.html" target="_blank"> 
+                  <button id="discover">discover</button></a></div>
+          <div id="discover-now"></div>
+        <h3 class="product-title">sneakerhead haven</h3>
+</div>
+    
+<div class="banner-2">
+    <h2> air-jordan</h2>
+  <p><a href="#">shop</a> </p> 
+</div>
+
+<div class="banner-3">
+    <h2>casual fashion</h2>
+    <p><a href="#">shop</a> </p>
+</div>
+    <div class="product-title2"><p>more to explore</p></div>
+    
+    <div class="products-container">   
+    <div class="products">
```

Comment:
@brownfrosamurai  Ensure that you use proper indentation. For instance, your closing div indentation should be the same as the indentation of the starting div... and others like that.
But this is good work, well done!!!